tests/task_queue_consumer/task_queue_consumer_test.py

```diff
@@ -28,11 +28,14 @@ def ssm_return_vals(using_private):
     ssm_prefix = f"/{app_name}/{stage}"
     return {
         'Parameters': [
-            {"Name": f"{ssm_prefix}/vpc/using_private_subnets", "Value": "true" if using_private else "false"},
-            {"Name": f"{ssm_prefix}/tasks/{task_name}/security_group/id", "Value": "sg-123"},
+            {"Name": f"{ssm_prefix}/vpc/using_private_subnets",
```

Comment:
Formatting only changes adding noise, what is your line length setup? Can we avoid these please?